web-transport-proto/src/settings.rs

```diff
@@ -20,6 +21,36 @@ impl Setting {
     pub fn encode<B: BufMut>(&self, buf: &mut B) {
         self.0.encode(buf)
     }
+
+    // Reference : https://datatracker.ietf.org/doc/html/rfc9114#section-7.2.4.1
+    pub fn is_grease(&self) -> bool {
+        let val = self.0.into_inner();
+        if val < 0x21 {
+            return false;
+        }
+
+        (val - 0x21) % 0x1f == 0
+    }
+}
+
+impl Debug for Setting {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match *self {
+            Setting::QPACK_MAX_TABLE_CAPACITY => write!(f, "QPACK_MAX_TABLE_CAPACITY"),
```

Comment:
We should eventually make `Setting` an enum to avoid this boilerplate. But it's good enough for now.